### PR TITLE
Add simplified Chinese translation (zh_CN)

### PR DESCRIPTION
**Pull request by Lafudoci**

### CODE REVIEW

File: monero-wallet-gui.pro [hunk 1]
```diff
@@ -291,7 +291,8 @@ TRANSLATIONS =  \ # English is default language, no explicit translation file
                 $$PWD/translations/monero-core_nl.ts \ # Dutch
                 $$PWD/translations/monero-core_pl.ts \ # Polish
                 $$PWD/translations/monero-core_ru.ts \ # Russian
-                $$PWD/translations/monero-core_zh.ts \ # Chinese (Traditional)
+                $$PWD/translations/monero-core_zh_CN.ts \ # Chinese (Simplified-China)
+                $$PWD/translations/monero-core_zh_TW.ts \ # Chinese (Traditional-Taiwan)
```
Comment (Jaqueeee) [inline monero-wallet-gui.pro]: Filenames
monero-core_zh-cn.ts
monero-core_zh-tw.ts

File: lang/languages.xml [hunk 2]
```diff
@@ -29,5 +29,6 @@ List of available languages for your wallet's seed:
         <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
         <!-- <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Portuguese" flag="/lang/flags/brazil.png" qs="none"/> -->
         <!-- <language display_name="русский язык" locale="ru_RU" wallet_language="Russian" flag="/lang/flags/russia.png" qs="none"/> -->
+        <language display_name="简体中文 (中国)" locale="zh_CN" wallet_language="English" flag="/lang/flags/china.png" qs="none"/>
         <language display_name="繁體中文 (台灣)" locale="zh_TW" wallet_language="English" flag="/lang/flags/taiwan.png" qs="none"/>
```
Comment (Jaqueeee) [inline lang/languages.xml]: Locale string should be:
zh-cn_CN
zh-tw_CN

File: monero-wallet-gui.pro (hunk 1 above)
Comment (Lafudoci) [inline monero-wallet-gui.pro]: done!

File: lang/languages.xml (hunk 2 above)
Comment (Lafudoci) [inline lang/languages.xml]: done!

**Comment by NanoAkron** [comment]
You probably want to use the mainland Chinese flag for simplified Chinese

**Comment by Lafudoci** [comment]
NanoAkron : sure, I pointed it to china.png in languages.xml.

**Pull request merged**